config-linux.md

```diff
@@ -509,10 +508,9 @@ The actions, architectures, and operators are strings that match the definitions
 The following parameters can be specified to set up seccomp:
 
 * **`defaultAction`** *(string, REQUIRED)* - the default action for seccomp. Allowed values are the same as `syscalls[].action`.
-
 * **`architectures`** *(array of strings, OPTIONAL)* - the architecture used for system calls.
-    A valid list of constants as of libseccomp v2.3.2 is shown below.
 
+    A valid list of constants as of libseccomp v2.3.2 is shown below.
```

Comment:
I think you want to keep this line where it was (as part of the initial paragraph, with a blank line after it separating the sub-list).

Reply:
updated.